Content.Server/Doors/Systems/DoorSystem.cs

```diff
@@ -91,124 +92,128 @@ protected override void PlaySound(EntityUid uid, string sound, AudioParams audio
             filter.RemoveWhereAttachedEntity(e => e == predictingPlayer);
         }
 
-        // send the sound to players.
-        SoundSystem.Play(filter, sound, uid, audioParams);
-    }
-
-    #region DoAfters
-    /// <summary>
-    ///     Weld or pry open a door.
-    /// </summary>
-    private void OnInteractUsing(EntityUid uid, DoorComponent door, InteractUsingEvent args)
```

Comment:
There's a whole bunch of whitespace changes. I think you just accidentally de-indented a large chunk of this file.